content/docs/rfcs/13/README.md

```diff
@@ -85,7 +85,8 @@ To perform pagination, each `WakuMessage` stored at a node running the `WakuStor
 
 RPC call to query historical messages.
 
-- The `topics` field MUST indicate the list of topics to query.
+- The `topics` field MUST indicate the list of topics to query. 
+  The items in this list correspond to the `contentTopic` of the queried Waku messages (see [14/WAKU-MESSAGE](../14)).
```

Comment:
```suggestion
  The items in this list correspond to the `contentTopic` of the queried Waku messages (see [14/WAKU-MESSAGE](/spec/14)).
```
This should be safer

Reply:
I tried `/spec/14` before but it seemed to be a broken link, that is why I changed it to `../`

Reply:
I think only when previewing it in Github, for https://rfc.vac/dev/spec/14 the relative path should work and be more canonical